Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/callback/openSAML/SAMLAssertionBuilderFactoryImplTest.java

```diff
@@ -35,9 +35,9 @@
     @Test
     public void getBuilderTest() {
```

Comment:
Should this be two separate tests, since it's two separate conditions with two separate asserts?  Also, is there an associated negative path test (i.e., testing the behavior when passing in a random String or a null value)?